style.css

```diff
@@ -174,3 +176,72 @@ button:active {
   color: #6070ff;
   padding: 4px 12px;
 }
+
+/* Contact Form Style */
+.my-form {
```

Comment:
- [ ] Can we please use `flex-box` to layout the content inside the form section. we need to use flex-box whenever we need to lay a collection of items out in one direction or another. As you layout your items you want to control the dimensions of the items in that one dimension or control the spacing between items